lib/ordered_list.rb

```diff
@@ -0,0 +1,37 @@
+class Ols
+
+  def ordered_list_tag_open(input)
+    input.each_with_index{|x, idx| x[0].to_i > 0 && input[idx-1] == "\n\n" ? input.insert(idx, "<ol>\n") : x}
+  end
+        
+  def ordered_list_tag_close(input)
+    input.each_with_index do |x, idx|
+      x[0].to_i > 0 && input[idx-2][0].to_i > 0 && (input[idx+2].nil? ||input[idx+2][0] == 0 || input[idx+1] == "\n\n") ? input.insert(idx+1,"\n</ol>" ) : x
```

Comment:
what is `x`?

what is `input[idx - 2][0].to_i`?

Also please make ample use of spacing in arrays as well as mathematical operations 😄